ios/PluginTests/ScreenSizePluginTests.swift

```diff
@@ -0,0 +1,25 @@
+import XCTest
+@testable import Plugin
+
+class ScreenSizeTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+
+    func testEcho() {
+        // This is an example of a functional test case for a plugin.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+
+        let implementation = ScreenSize()
+        let value = "Hello, World!"
+        let result = implementation.echo(value)
```

Comment:
should be replaced by ppi method

Reply:
Test for devicePPI()-method is now in place.